cfme/base/ui.py

```diff
@@ -1095,6 +1096,19 @@ def step(self, *args, **kwargs):
         self.prerequisite_view.advanced.select()
 
 
+@navigator.register(Region, "Utilts")
```

Comment:
`Utilts` stands for ` Utilization Trend Summary`; I took this form developers id formats like `miq_chart_parent_utilts`

Reply:
Gotta say, I'm not a fan of this abbreviation, its not clear what its for. I think some more verbosity is worthwhile in this case. We have destinations like `'DetailsFromProvider'`, how about `'UtilTrendSummary'` so that its very clear what the destination is, without being too long.